lib/publiccloud/gcp_client.pm

```diff
@@ -37,7 +37,7 @@ sub init {
     my ($self) = @_;
     # For now we support Vault and the credentials-microservice. Vault will be removed after a certain transition period
     if (get_var('PUBLIC_CLOUD_CREDENTIALS_URL')) {
-        my $data = get_credentials(CREDENTIALS_FILE);
+        my $data = get_credentials('gce.json', CREDENTIALS_FILE);
```

Comment:
Why is this `CREDENTIALS_FILE` missing on the other clients?

Reply:
Because it's not needed. Only Google needs a phyisical file to read the credentials from, the others are use environment variables.